notebooks/proj_sys_eng/sitcom_integration/LVV-T2219-ComCam_Interface_to_SAL.ipynb

```diff
@@ -2194,27 +1272,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 80,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "RemoteCommand(CCCamera, 0, stop)"
-      ]
-     },
-     "execution_count": 80,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "comcam.rem.cccamera.cmd_stop"
+    "comcam.rem.cccamera.cmd_stop.set_start()"
```

Comment:
I guess this requires an `await`?

Reply:
Thank you for your careful attention. `await comcam.rem.cccamera.cmd_stop.set_start()` gives an error, whereas the same command without the `await` doesn't when run in the TTS, it's been decided to leave it without the `await` until we run it in the summit. It might be that the command is not yet implemented, as it was mentioned in conversations with Kevin and Tony.